composer.json

```diff
@@ -15,6 +15,7 @@
         "ext-gmp": "*",
         "ext-intl": "*",
         "ext-redis": "*",
+        "ext-tokenizer": "*",
```

Comment:
the tokenizer extension is already required by laravel/framework, so I won't add it here
```suggestion
```